docs/src/pages/fi-FI/blog/island-architecture.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: ~/layouts/Main.astro
+title: Saariarkkitehtuuri
+draft: true
+---
+
+<!--
+    @aFuzzyBear: I have been spending most the day learning more about Island Architecture, wrote plenty of notes, listened to Fred K Schott's interview on Speakeasy(https://www.youtube.com/watch?v=mgkwZqVkrwo) and the interview with Jason Lengstrof (https://www.youtube.com/watch?v=z15YLsLMtu4)
```

Comment:
This page isn't public! It was a work in progress that someone wanted to get merged in as a part of our original docs push, but it never made it onto the site. Safe to remove from your translation.